util/cron/common-slurm-gasnet-cray-cs.bash

```diff
@@ -22,5 +22,4 @@ nightly_args="${nightly_args} -no-buildcheck"
 export CHPL_TARGET_CPU=native
 export GASNET_PHYSMEM_MAX=16G
 export GASNET_PHYSMEM_NOPROBE=1
-export GASNET_ODP_VERBOSE=0
```

Comment:
Will this not have to be stuck back in here when we eventually re-enable ibv-odp after the bug is fixed? If so, maybe just leave it for now?

Reply:
Looking at the history, I believe `GASNET_ODP_VERBOSE` was really only needed on our older FDR IB systems where gasnet detected ODP support might be available at configure time, but at execution time saw it wasn't actually functional. On the current HDR IB system I actually think this isn't needed at all since there is full ODP support. 

And as far as re-enabling ibv-odp -- I actually suspect we won't do that unless there is support for using ODP over firehose. i.e. a mode that lets you just use on-demand-paging instead of implementing it in software. This is requested in https://upc-bugs.lbl.gov/bugzilla/show_bug.cgi?id=3941, but I suspect early testing of the prototype resulted in hangs.

Reply:
Got it -- sounds good, then.